bootstrap/apollo-env.sh

```diff
@@ -14,3 +16,10 @@ if [[ -z "${color_start-}" ]]; then
   declare -r color_norm="${color_start}0m"
 fi
 
```

Comment:
The levels just gives you a centralized way to manage different combinations. The way it is right now once I set APOLLO_LOG there is no way to silence the Apollo log but disabling each indivdually (2 at the moment, we might want add more like packer/vagrant). So I'd definitely like an APOLLO_LOG=0
Then if you unset APOLLO_LOG you can set differents levels via ANSIBLE_LOG or per tool.